includes/class-reader-activation.php

```diff
@@ -1150,8 +1150,13 @@ public static function send_verification_email( $user ) {
 		$subject = __( 'Please verify your account', 'newspack' );
 
 		/* translators: %s: Site title. */
-		$message  = sprintf( __( 'Welcome to %s!', 'newspack' ), $blogname ) . "\r\n\r\n";
```

Comment:
We could do a generic greeting like `Hello, {displayname}!`, like in newsletters verification email.

Reply:
Generally at this point we don't have a proper display name since we're just asking for email, so we could do something like `Hello, {email address}` or something more generic like `Hi there!`

Reply:
The display name at this point is the email username, so it would be `john` for `john@doe.com`. But I agree we can use something more generic.

Reply:
I made this `Hi there!` in 4007076.